thirdparty/download-thirdparty.sh

```diff
@@ -423,6 +423,15 @@ else
     echo "$AWS_SDK_CPP_SOURCE not patched"
 fi
 
+cd $TP_SOURCE_DIR/$AWS_SDK_CPP_SOURCE
+if [ $AWS_SDK_CPP_SOURCE = "aws-sdk-cpp-1.11.267" ]; then
```

Comment:
Can you remove `aws-sdk-cpp-1.10.36` and `aws-sdk-cpp-1.9.179`, I think we don't need it anymore.

Remove relavant patch either.

If you are convenient.

Reply:
> Can you remove `aws-sdk-cpp-1.10.36` and `aws-sdk-cpp-1.9.179`, I think we don't need it anymore.
> 
> Remove relavant patch either.
> 
> If you are convenient.

agree, I will do it